### PR TITLE
Replace views grid table style with CSS grid property.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2811

### CODE REVIEW

File: core/modules/views/templates/views.theme.inc [hunk 1]
```diff
@@ -680,45 +679,27 @@ function template_preprocess_views_view_table(&$variables) {
  */
 function template_preprocess_views_view_grid(&$variables) {
   $view     = $variables['view'];
+  $rows     = $variables['rows'];
   $result   = $view->result;
-  $options  = $view->style_plugin->options;
-  $handler  = $view->style_plugin;
+  $style    = $view->style_plugin;
+  $options  = $style->options;
```
Comment (klonos) [inline core/modules/views/templates/views.theme.inc:685]: I know this is in order to make this more pretty/readable, but I think that it is against our coding standards.
```suggestion
  $options = $style->options;
```

File: core/modules/views/templates/views.theme.inc [hunk 2]
```diff
@@ -680,45 +679,27 @@ function template_preprocess_views_view_table(&$variables) {
  */
 function template_preprocess_views_view_grid(&$variables) {
   $view     = $variables['view'];
```
Comment (klonos) [inline core/modules/views/templates/views.theme.inc:681]:
```suggestion
  $view = $variables['view'];
```

File: core/modules/views/templates/views.theme.inc [hunk 3]
```diff
@@ -680,45 +679,27 @@ function template_preprocess_views_view_table(&$variables) {
...
+  $rows     = $variables['rows'];
```
Comment (klonos) [inline core/modules/views/templates/views.theme.inc:682]:
```suggestion
  $rows = $variables['rows'];
```

File: core/modules/views/templates/views.theme.inc [hunk 4]
```diff
@@ -680,45 +679,27 @@ function template_preprocess_views_view_table(&$variables) {
...
   $result   = $view->result;
```
Comment (klonos) [inline core/modules/views/templates/views.theme.inc:683]:
```suggestion
  $result = $view->result;
```

File: core/modules/views/templates/views.theme.inc [hunk 5]
```diff
@@ -680,45 +679,27 @@ function template_preprocess_views_view_table(&$variables) {
...
-  $options  = $view->style_plugin->options;
-  $handler  = $view->style_plugin;
+  $style    = $view->style_plugin;
```
Comment (klonos) [inline core/modules/views/templates/views.theme.inc:684]:
```suggestion
  $style = $view->style_plugin;
```

**Comment by herbdool** [comment]
The vertical alignment setting doesn't work with this PR. It doesn't seem to reorder them vertically and it is only adding classes to the first three items. (I'm also not clear on why each item is considered a "row" in the css classes though not a big deal.)

screenshot from 2018-11-03 15-02-17

**Comment by quicksketch** [comment]
Replaced by https://github.com/backdrop/backdrop/pull/2447.

**Pull request closed without merging**